assets/training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/predict.py

```diff
@@ -202,6 +203,16 @@ def predict(
     logging.info(f"Using device: {device} for the inference")
 
     result = []
+
+    pipe = pipeline(
+        task=task,
+        model=model,
+        tokenizer=tokenizer.tokenizer,
+        feature_extractor=tokenizer.feature_extractor,
+        chunk_length_s=30,
```

Comment:
Does this chunk length work with every input length? Should we take the chunklength as an optional parameter from input, as a part of kwargs, so that even if 30s gives erroneous output, the user can change the chunk length.